arches_lingo/src/arches_lingo/App.vue

```diff
@@ -64,8 +74,9 @@ router.beforeEach(async (to, _from, next) => {
         <PageHeader v-if="route.meta.shouldShowNavigation" />
         <div style="display: flex; flex: auto; flex-direction: row">
             <SideNav v-if="route.meta.shouldShowNavigation" />
-            <div style="margin: 1rem; flex: auto">
-                <RouterView />
+            <div style="flex: auto">
+                <HierarchySplitter v-if="route.meta.shouldShowHierarchy" />
```

Comment:
Good solution! 👍

Reply:
I agree this is better--thanks for encouraging me down this path!